base/dft.jl

```diff
@@ -580,7 +580,7 @@ module FFTW
     """
     function plan_r2r end
 
-    Base.USE_GPL_LIBS && include(joinpath("fft", "FFTW.jl"))
+    (Base.USE_GPL_LIBS || Base.USE_INTEL_MKL_FFT) && include(joinpath("fft", "FFTW.jl"))
```

Comment:
Of course there aren't that many fftw vendors out there, so perhaps no need for overkill here either.